build_projects/shared-build-targets-utils/Utils/Monikers.cs

```diff
@@ -93,6 +71,11 @@ public static string GetSdkDebianPackageName(BuildTargetContext c)
             return $"{packagePrefix}-dev-{nugetVersion}";
         }
 
+        public static string GetDebianHostFxrPackageName(BuildTargetContext c)
```

Comment:
this is not correct any longer. Please take the latest of this method that I have merged in core-setup